internal/pkg/config/v1/config_test.go

```diff
@@ -26,17 +26,21 @@ import (
 const (
 	authTokenPlaceholder        = "AUTH_TOKEN_PLACEHOLDER"
 	authRefreshTokenPlaceholder = "AUTH_REFRESH_TOKEN_PLACEHOLDER"
-	saltPlaceholder             = "SALT_PLACEHOLDER"
-	noncePlaceholder            = "NONCE_PLACEHOLDER"
+	saltPlaceholder             = "SALT_TOKEN_PLACEHOLDER"
+	noncePlaceholder            = "NONCE_TOKEN_PLACEHOLDER"
+	apiSecretPlaceholder        = "API_SECRET_PLACEHOLDER"
+	apiSaltPlaceholder          = "c2FsdHBsYWNlaG9sZGVy"
+	apiNoncePlaceholder         = "bm9uY2VwbGFjZWhvbGRlcg=="
```

Comment:
Should we be using actual values here, or can we use UPPERCASE_STRINGS like the other placeholder constants?

Reply:
somehow they're illegal in base64, so i used the encoded version of them